decision/decision_pre_triggers_and_alias.cwt

```diff
@@ -79,7 +86,16 @@ single_alias[filtered_decisions] = {
     ### Factors for an AI character to take the decision (1 = 100% chance)
     ai_will_do = single_alias_right[decision_ai_will_do_clause]
 
-    # TODO: ai_accceptance
+    ## cardinality = 0..1
+    ## replace_scope = { this = character root = character from = character }
+    ai_acceptance = {
+        factor = int
+        yes_threshold = int
+        show_maybe_for_human = bool
```

Comment:
Defaults? Ranges for the above? Should keep a #TODO comment above if you're unwilling to do them now :)